### DISCUSSION TITLE
Can’t launch Neo Geo/Amiga Roma

### DISCUSSION THREAD

**Discussion by rromney1** [eduardofilo/RG350_adam_image#142] [General]
I have downloaded the updated version of Adam image for my RG350 but I can't get the Neo Geo Roms to work. I downloaded the Neo Geo bios and still nothing. The Amiga also the  same thing it keep asking for kickstart and I have downloaded all kickstart Roms but still nothing any advice to help me? Please and Thank you!!!!!

**Comment by eduardofilo** [comment]
Neo Geo is a bit different from other systems in terms of which BIOS to install and where to put it. On the one hand, it is a zip file with several ROMs inside and on the other it must be in the same directory as the ROMs, and not in the `bios` directory as in most other systems. There must also be a correspondence between its content and some of the settings in Options (`Neo Geo Mode`, `BIOS`). This is why I did not put an MD5 hash on the table. You already know that I can't tell you where to find the BIOS, but the solution is to try searching to find the one that it works.

The situation with the Amiga system is slightly simpler, but also more difficult than average. Review what is said in discussion #136.
